### PR TITLE
Pasindu production - Conflicts checked

### PR DESCRIPTION
**Pull request by PasinduVinod**

### CODE REVIEW

File: routes/STDAvailableSubmissions.js [hunk 1]
```diff
@@ -24,7 +24,7 @@ router.route("/addSubmission").post(addSubmission)
 
 router.route("/deleteSubmission").delete(DeleteSubmission)
 
-router.route("/viewSpecificSubmission").get(viewSpecificSubmission)
+//router.route("/viewSpecificSubmission").get(viewspecificSubmission)
```
Comment (shas99) [inline routes/STDAvailableSubmissions.js:27]: Remove this comment

**Review by shas99 — CHANGES_REQUESTED** [review]
Please uncomment line 27 in STDAvailableSubmissions @PasinduVinod

**Review by PasinduVinod — COMMENTED** [review]
line 27 uncommented

**Comment by PasinduVinod** [comment]
closed

**Comment by shas99** [comment]
Files changed

File: routes/STDAvailableSubmissions.js [hunk 2]
```diff
@@ -24,7 +24,7 @@ router.route("/addSubmission").post(addSubmission)
 
 router.route("/deleteSubmission").delete(DeleteSubmission)
 
-router.route("/viewSpecificSubmission").get(viewSpecificSubmission)
+router.route("/viewSpecificSubmission").get(viewspecificSubmission)
```
Comment (shas99) [inline routes/STDAvailableSubmissions.js:27]: Make "viewspecificSubmission" inside getting method to "viewSpecificSubmission"

**Review by shas99 — APPROVED** [review]
@shas99 LGTM

**Pull request merged**